bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java

```diff
@@ -91,7 +101,9 @@ public void onFinished() {
             }
         });
 
-        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
+        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
+        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
```

Comment:
Correct. 

I don't mind if we add it, but given it is part of the request it didn't seem necessary. 

Sent from my iPhone

> On 9 Aug 2016, at 08:25, Markus Rathgeb <notifications@github.com> wrote:
> 
> In bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/discovery/DiscoveryResource.java:
> 
> > @@ -91,7 +101,9 @@ public void onFinished() {
> >              }
> >          });
> >  
> > -        return Response.ok(discoveryServiceRegistry.getMaxScanTimeout(bindingId)).build();
> > +        DiscoveryServiceDTO dto = new DiscoveryServiceDTO();
> > +        dto.timeout = discoveryServiceRegistry.getMaxScanTimeout(bindingId);
> I assume you miss the id attribute by intention because it the same as given by the parameter on request. Correct?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>